myapp/app/controllers/tasks_controller.rb

```diff
@@ -23,7 +23,7 @@ def edit
 
   def create
     @task = Task.new(task_params)
-    @task.user_id = 1 # because user must exist
+    @task.user_id = current_user['id']
     @task.user_type = 'Task' # user_type is needed because of NOT NULL
```

Comment:
[nitpick] Not related to this. But why is `user_type` a task? Shouldn't `user_type` be like normal, admin, or something like that?

Reply:
that was generated automatically due to use `polymorphic: true`. I'll migrate it.